api-common/src/main/kotlin/io/qalipsis/api/meters/StatisticsFactory.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2024 AERIS IT Solutions GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package io.qalipsis.api.meters
+
+import java.time.Instant
+
+/**
+ * Module to manage global meter statistics, generated by a campaign.
+ *
+ * @author Francisca Eze
+ */
+interface StatisticsFactory {
```

Comment:
1. Let's move this interface to the factory module, since all the implementations will be there.
2. Let's rename the interface "MeterFactory". StatisticsFactory is not good at all.

Reply:
Some other name options 
- MetricsManager
- MeterManager
- StatisticsRegistry
- MeterMonitoringRegistry

I think I love the MetricsManager and StatisticsRegistry. We could have GlobalStatisticsRegistry or GlobalMetricsManager

Reply:
so let's use MetricsManager.

Reply:
> Let's move this interface to the factory module, since all the implementations will be there.

Please where is the factory module. I didn't seen any module named factory in the API, or do you mean I should create one?

Reply:
I mean it should be moved to the oss project, in the factory module. There is no need to make it part of the public API.

Reply:
Ohhh okay